### PR TITLE
Added 3 quotes

### PR DESCRIPTION
**Pull request by sarveshh**
## ✅ New Quote Checklist
- [x] The quote(s) added are in alphabetical order by author's first name and not ❌ duplicates.
- [x] The quote(s) added are about **learning a foreign language**.
- [x] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [x] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -755,6 +763,10 @@
     "author": "Turkish Proverb",
     "sentence": "One who speaks only one language is one person, but one who speaks two languages is two people."
   },
+  {
+    "author": "Turkish Proverb",
```
Comment (Kvaibhav01) [inline server/data/quotes.json:767]: This is a [duplicate quote](https://github.com/jayehernandez/letra-extension/blob/08226428a914bb75266e566bad4ad86056bab92e/server/data/quotes.json#L755). Please remove it.

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Thanks for the PR. Update the necessary changes.

File: server/data/quotes.json (hunk 1 above)
Comment (sarveshh) [reply server/data/quotes.json]: Hey @Kvaibhav01 apology from my side. I have done the necessary changes and committed them. Please review. ThankYou.

Comment (Kvaibhav01) [reply server/data/quotes.json]: Looks good now. @jayehernandez will take care of this PR now for further actions.

**Review by jayehernandez — APPROVED** [review]
Thank you for contributing to Letra ☀️, @sarveshh!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Pull request merged**